### PR TITLE
Landscape self service download server

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Added "Your next steps with Ubuntu Server" section and added the list in Ubuntu Server 16.04.1 LTS for OpenStack.

## QA
- Pull code
- Check the code is ok
- Run `./run` and go to http://0.0.0.0:8001/download/server
- Check that the page works on all sizes and looks like the screenshot

Copy doc: https://docs.google.com/document/d/1FFQQUtzFGH88hA30qnZm-3pwVqcVl9UYyrNT4qQWrSM/edit?pli=1

### CODE REVIEW

**Comment by anthonydillon** [comment]
[Update /download/server for landscape-self-service](https://trello.com/c/M7zK4vqa/2988-update-download-server-for-landscape-self-service)

File: templates/download/server/index.html [hunk 1]
```diff
@@ -46,23 +46,125 @@
...
+            <h2>Your next steps with Ubuntu Server</h2>
+            <p>
+                Use Ubuntu&rsquo;s tools to help you provision and manage
+                your servers
+            </p>
+        </div>
+        <div class="equal-height">
+            <div class="four-col box">
+                <h3>
+                    Securely patch your kernels without rebooting with the
```
Comment (grahambancroft) [inline templates/download/server/index.html]: This live patch item should be removed as per copy doc https://docs.google.com/a/canonical.com/document/d/1FFQQUtzFGH88hA30qnZm-3pwVqcVl9UYyrNT4qQWrSM/edit?pli=1&disco=AAAABIz7uXU

**Comment by grahambancroft** [comment]
@anthonydillon Will this need design approval?

File: templates/download/server/index.html (hunk 1 above)
Comment (anthonydillon) [inline templates/download/server/index.html]: Sorry, I should have mentioned. I plan to get this branch all landed with the docs and then have follow-ups to remove the live patch stuff. So I can just cherry pick to commit when it comes to going live with the live patch stuff.

**Comment by anthonydillon** [comment]
No design review needed as this page will change in a follow-up branch.

**Comment by grahambancroft** [comment]
Lgtm 👍

**Pull request merged**